algorithms/src/snark/varuna/ahp/indexer/circuit_info.rs

```diff
@@ -19,8 +19,7 @@ use snarkvm_utilities::{serialize::*, ToBytes};
 /// Information about the circuit, including the field of definition, the number of
 /// variables, the number of constraints, and the maximum number of non-zero
 /// entries in any of the constraint matrices.
-#[allow(clippy::derive_partial_eq_without_eq)]
-#[derive(Copy, Clone, Debug, PartialEq, Eq, Ord, PartialOrd, CanonicalSerialize, CanonicalDeserialize)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, CanonicalSerialize, CanonicalDeserialize)]
```

Comment:
%100 the ord wasn't needed here since all the members are public and directly comparable.